packages/core/src/external/carequality/ihe-gateway/outbound-result-poller-lambda.ts

```diff
@@ -28,67 +28,80 @@ export class OutboundResultPollerLambda extends OutboundResultPoller {
     this.docRetrievalLambdaName = docRetrievalLambdaName;
   }
 
-  isPDEnabled(): boolean {
-    return (this.patientDiscoveryLambdaName ?? "").trim().length > 0;
+  isPDEnabled(
+    lambdaName: string | undefined = this.patientDiscoveryLambdaName
+  ): lambdaName is string {
+    return (lambdaName ?? "").trim().length > 0;
   }
```

Comment:
This allows us to code w/ the lambda name below w/o having to have the non-null-assertion